osconfig_tests/utils/utils.go

```diff
@@ -21,12 +21,24 @@ import (
 )
 
 var (
-	// InstallOSConfigDeb installs the osconfig agent on deb based systems.
-	InstallOSConfigDeb = `echo 'deb http://packages.cloud.google.com/apt google-osconfig-agent-stretch-unstable main' >> /etc/apt/sources.list
+	// TODO: the startup script installs the osconfig agent and then keeps querying
```

Comment:
i had this in my mind, but i wanted to have a more generic solution for running multiple startup scripts keeping in mind that we might have to change this when we write tests for patching( i am not sure what would change, though)
now i think, instead of having a generic solution to run multiple startup scripts, how about having a generic solution to create multiple startup scripts, like an interface that each test-suite can use and implement their own way of creating a single start-up script.

Reply:
i left the startupscripts as it is and moved the query string to specific testSetup and package manager. this also allow us to have specific query string based on the package manager and the package that is being used for the test.